install.sh

```diff
@@ -3,12 +3,17 @@
 case $OSTYPE in
   darwin*)
     cargo build --release
-    mkdir -p $HOME/Applications
+    mkdir -p "$HOME/Applications"
     APP=$HOME/Applications/WezTerm.app
-    rm -rf $APP
-    cp -r assets/macos/WezTerm.app $APP && \
-      cp target/release/wezterm $APP && \
-        echo "Installed to $APP"
+
+    # compile xib
+    BASE_LPROJ="$APP/Contents/Resources/Base.lproj/"
+
+    rm -rf "$APP"
+    cp -r assets/macos/WezTerm.app "$APP" && \
+      cp target/release/wezterm "$APP" && \
+        ibtool --compile "$BASE_LPROJ/menu.nib" "$BASE_LPROJ/menu.xib" && \
```

Comment:
I had to run `sudo xcode-select -s /Applications/Xcode_10.2.app` to get ibtool to run; it doesn't run with the CLI only flavor of the developer tools.  I think that is probably fine, but its worth checking that it will run in the `ci/deploy.sh` script which packages up the .zip file that is produced when I push a tag.

I think we need to add this same step to that script, and should probably change the `.travis.yml` to always run `ci/deploy.sh` in the `script` section, rather than in the `before_deploy` section so that it gets tested in PRs too.